docs/az/fluttertimertutorial.md

```diff
@@ -0,0 +1,1031 @@
+# Flutter Timer Tutorial
+
+![beginner](https://img.shields.io/badge/level-beginner-green.svg)
+
+> In the following tutorial we’re going to cover how to build a timer application using the bloc library. The finished application should look like this:
+
+![demo](./assets/gifs/flutter_timer.gif)
+
+## Setup
+
+We’ll start off by creating a brand new Flutter project
+
+```sh
+flutter create flutter_timer
```

Comment:
can you please rebase? I've pushed changes to master to reuse these code snippets throughout all the docs. Thanks 👍